packages/grid/src/GridUtils.test.ts

```diff
@@ -155,14 +168,14 @@ describe('floating item checks', () => {
 
 describe('start/end range adjustment in one dimension', () => {
   function testRange(
-    start,
-    end,
-    movedItems = [],
+    start: GridRangeIndex,
+    end: GridRangeIndex,
+    movedItems: MoveOperation[] = [],
     expectedResult = [[start, end]]
   ) {
     expect(
       GridUtils.getModelRangeIndexes(start, end, movedItems).sort(
-        (a, b) => a[0] - b[0]
+        (a, b) => (a as AxisRange)[0] - (b as AxisRange)[0]
```

Comment:
The function returns a `GridAxisRange`, which could have `null` values. This casts it to `AxisRange` so no `null` values

Reply:
The cast doesn't actually mean there's no `null` values though, so wouldn't it be possible that you end up w/ `null` values since the function returns a possible `null`?

Reply:
Well, I guess, in which case the test would fail, since we're not expecting null here... do I need to rewrite it to explicitly check for null, or is casting with what's expected of the test knowing that it'll fail the test if it's null is good enough?

Reply:
If It will fail the test I'd say that's good enough? Looks like the sort would treat `null` as `0` index effectively

Reply:
Yea it will fail